src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -46,6 +46,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- This is needed for the Appx targets -->
+    <DefaultLanguage Condition="'$(DefaultLanguage)' == ''">en-US</DefaultLanguage>
+    
+    <!-- Many of the targets use the VS Version in the path -->
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &lt; '14.0' ">15.0</VisualStudioVersion>
```

Comment:
I have Xamarin Android installed on Linux, so I hope that the XA TFMs aren't tied to any OS.